contrib/destination-starrocks/src/test-integration/java/io/airbyte/integrations/destination/starrocks/StarrocksDestinationAcceptanceTest.java

```diff
@@ -12,6 +12,10 @@
 import io.airbyte.integrations.standardtest.destination.DestinationAcceptanceTest;
 import java.io.IOException;
 import java.nio.file.Paths;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
```

Comment:
Unnecessary imports?

Reply:
Imports are used in the file. The code wasn't compiling before.